sys/arm64/arm64/support.S

```diff
@@ -218,6 +238,26 @@ EENTRY(suword64)
 EEND(suword64)
```

Comment:
All the fu* and su* functions need to be using the cap arg (`[c0]`) rather than the integer pointer arg (`[x0]`)